modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/NewQuestion.es.js

```diff
@@ -120,20 +120,26 @@ export default withRouter(
 			getSectionBySectionTitle,
 		]);
 
-		const processError = (error) => {
-			if (error.message && error.message.includes('AssetTagException')) {
-				error.message = lang.sub(
-					Liferay.Language.get(
-						'the-x-cannot-contain-the-following-invalid-characters-x'
+		const processError = (tag) => {
+			const invalidCharacters = /.*[-|+|=|_|!|$|&|@|#|%|?|()|/\\|[\]|><||{}|:|;|,|.|`|'|"|*|~|^|\n|\r]/g;
+			if (invalidCharacters.test(tag)) {
+				const error = {
+					message: lang.sub(
+						Liferay.Language.get(
+							'the-x-cannot-contain-the-following-invalid-characters-x'
+						),
+						[
+							Liferay.Language.get('tag-name'),
+							' & \' $ ( ) ! @ \\\\ ] } : . , = > / < \\n [ {  | - + # ` ? \\" \\r ; / * ^ ~',
+						]
 					),
-					[
-						'Tag',
-						' & \' @ \\\\ ] } : , = > / < \\n [ {  | + # ` ? \\" \\r ; / * ~',
-					]
-				);
+				};
+				setError(error);
+
+				return false;
 			}
 
-			setError(error);
+			return true;
```

Comment:
@larissacribeiro this code worries me a little, let me explain:

Before, the code had the following statement: 

```js
error.message && error.message.includes('AssetTagException')
```

But checking the API Response, it no longer appears, which is a problem for us, because we don't know why the error happened in the first place.

Also, the new code check in any error circumstance for invalid characters, so if the API returns:

```json
{
  "errors": [
    {
      "message": "Exception while fetching data (/createMessageBoardSectionMessageBoardThread) : null",
      "locations": [],
      "extensions": {
        "exception": {
          "errno": 500
        },
        "code": "Internal Server Error",
        "classification": "DataFetchingException"
      }
    }
  ],
  "data": {
    "createMessageBoardSectionMessageBoardThread": null
  }
}
```

the message attribute with: `Exception while fetching data (/createMessageBoardSectionMessageBoardThread) : null`

will be passed to the `invalidCharacters.test(tag)` and identify it as invalid, because this text contains `(`, `)`, `/` .

Since the backend has the same options as invalid, the best alternative would be the backend send it throw the response, we capture and send a friendly error message to the user.

@nNilton is taking a look why the expectation message is not present on the API Response, then we can properly display the error message.